packages/remark-mdxs/readme.md

```diff
@@ -0,0 +1,55 @@
+# [remark][]-[mdx][]s
+
+[![Build Status][build-badge]][build]
+[![lerna][lerna-badge]][lerna]
+[![Join the community on Spectrum][spectrum-badge]][spectrum]
+
+> :warning: This project is currently in alpha
+
+[MDXs][] syntax support for [remark][].
+
+## Installation
+
+```sh
+npm install --save remark-mdxs
+```
+
```

Comment:
From a usage standpoint, is the thinking that you would use something like a separate webpack loader rule that only parses `.mdxs` files while handling `.mdx` files like any other?

Reply:
Yeah that's the idea 👍

Reply:
That seems like it will turn out to be kind of unfortunate. It means we can't delineate between MDX content and MDXS content when fetching from remote sources that use `text/markdown` content types (mdx still doesn't have it's own entry [in the mime-db](https://github.com/jshttp/mime-db/pull/136) and even if it did, we're going to want compat with markdown types from headless CMSs anyway).

Since every MDXs document is technically a valid MDX document, I'd much rather see a const export used to specify intent to be MDXs content.

Reply:
Hmm, that's a good point 🤔. I'd mostly been thinking about local fs. How do you think intent for MDXs should be expressed?

Something like?:

```jsx
export const MDXS = true
```

Reply:
yeah this is the approach I landed on as well. I think that the "first MDX file" in an MDXs document can be used as the "global exports" (compared to exports defined in subsequent "files"), so you'd end up with something like this where the first MDX doesn't have any content, but is just imports/exports.

```
// global export
export const MDXS = true
---
// file-local export
export const slideLayout = SomeComponent
# some slide
---
# another slide
```